caffe2/python/pybind_state_gpu.cc

```diff
@@ -40,6 +40,9 @@ void addCUDAGlobalMethods(py::module& m) {
   m.def("num_cuda_devices", &NumCudaDevices);
   m.def("get_cuda_version", &CudaVersion);
   m.def("get_cudnn_version", &cudnnCompiledVersion);
+  m.def("cudnn_convolution_fwd_algo_count", &CudnnConvolutionFwdAlgoCount);
```

Comment:
you can do `m.attr("cudnn_convolution_fwd_algo_count") = py::int_(CUDNN_CONVOLUTION_FWD_ALGO_COUNT);` to avoid making it a function in both python and c++

Reply:
Ok.